### PR TITLE
Add a nice introduction header to the README

### PR DESCRIPTION
**Pull request by sk1p**
It's always nice to have a good README (see issue #20)

## Contributor Checklist:

* [ ] I have added myself to [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)
* [ ] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
* [x] I have added/updated documentation for all user-facing changes
* [ ] I have added/updated test cases

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+LiberTEM-holo
+=============
+
+LiberTEM-holo is a python project for holographty reconstruction
```
Comment (sk1p) [inline README.rst:4]: Maybe add punctuation at the end?

Comment (sk1p) [reply README.rst:4]: Python should be capitalized!

```suggestion
LiberTEM-holo is a Python project for holographty reconstruction
```

Comment (uellue) [inline README.rst]:
```suggestion
LiberTEM-holo is a python project for holography reconstruction
```

File: README.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+LiberTEM-holo
+=============
+
+LiberTEM-holo is a Python project for holographty reconstruction
```
Comment (uellue) [inline README.rst:4]:
```suggestion
LiberTEM-holo is a Python project for holography reconstruction
```

**Pull request merged**